### PR TITLE
Pal 1034 add gitattributes

### PR DESCRIPTION
**Pull request by pd104923**
- Added gitattributes file
- Updated gitignore to include jvm example log and err files
- All files have been normalised based on the new git configs so line endings have changed for some of them.
- All Dockerfiles have had any `sed` operations removed

### CODE REVIEW

**Review by m78233 — DISMISSED** [review]

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
A lot of files have no changes, just line endings. Can you remove the files that have not been edited please

**Comment by pd104923** [comment]
> A lot of files have no changes, just line endings. Can you remove the files that have not been edited please

This is due to the changes that were made for this ticket. When doing a `git add --renormalize .` for the repo these files had their line endings updated.

File: .gitignore [hunk 1]
```diff
@@ -1,123 +1,125 @@
...
+##VSCODE default files that dont need to be tracked
+eclipse-formatter.xml
+.vscode/
+
+# Ignore H2 generated databases
+*.db
+
+# Ignore JVM example log/err files
+*.log
+*.err
```
Comment (developer01189998819991197253) [inline .gitignore:52]: are these changes needed?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Set the default behavior, in case people don't have core.autocrlf set
```
Comment (developer01189998819991197253) [inline .gitattributes:15]: add Palisade license header

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**